src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1957,12 +1954,128 @@ let validateAnnotationTable () =
                         [InteropLogging.Msg.create InteropLogging.Warning $"Table is not a valid ARC table / ISA table: {ex.Message}. The column {header} is not valid! It needs further inspection what causes the error.";
                         ])
                 else
-                    [InteropLogging.Msg.create InteropLogging.Warning $"The annotation table {excelTable.name} is valid"]
+                    []
 
             return messages
         }
     )
 
+let fillEmptyBuildingBlocks () =
+    Excel.run(fun context ->
+        promise {
+
+            let! msg = validateAnnotationTable ()
+            let mutable messages = msg
+
+            //When msg > 0 there is an error and we can skip the filling of the rows because the columns are not valid
+            if messages.Length > 0 then return messages
+            else 
+                let! excelTable = getActiveAnnotationTable context
```

Comment:
Must parse table a second time after `validateAnnotationTable`. Maybe have `validateAnnotationTable` have return the table if validated successful. This would require a new ouput using Result

```fsharp
let tryString (x: string) =
  if x.StartsWith "y" then
    Result.Ok true
  else
    Result.Error false


match tryString "yes" with
| Result.Ok x -> printfn "Ok: %A" x
| Result.Error x -> printfn "Error: %A" x
```

Reply:
use match case instead of if-else on line 1971

Reply:
fixed